src/algebra/polynomial/basic.lean

```diff
@@ -0,0 +1,52 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson, Jalex Stark.
+-/
+import data.polynomial
+open polynomial finset
+
+/-
+# Polynomials
+Eventually, much of data/polynomial.lean should land here.
+## Main results
+- We define two ring homs : polynomial R → R,
+- `coeff_zero_ring_hom` and `leading_coeff_monoid_hom`
+-/
+
+universes u w
+
+variables {R : Type u} {α : Type w}
+
+namespace polynomial
+
+section comm_semiring
+variables [comm_semiring R]
+
+/-- `polynomial.eval` bundled as a ring_hom -/
```

Comment:
Could you elaborate on this? Also if you want to use `ring_hom` or another Lean declaration name, please enclose it in backticks. (If you just refer to a ring hom informally, then it doesn't have to be in backticks.)